src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -49,70 +50,77 @@
 		"\tclear dropped experience",
 		"on break of diamond ore:",
 		"\tif tool of player = diamond pickaxe:",
-		"\t\tadd 100 to dropped experience"})
-@Since("2.1, 2.5.3 (block break event), 2.7 (experience change event)")
-@Events({"experience spawn", "break / mine", "experience change"})
+		"\t\tadd 100 to dropped experience",
+		"on breed:",
+		"\tevent-father is a cow",
```

Comment:
Are you sure `event-father` is a thing?

Reply:
Good catch, I believe this was my original plan before I decided using `breeding father` was best